alerts/honeycomb.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+from lib.alerttask import AlertTask
+from query_models import SearchQuery, TermMatch
+import re
+
+
+class AlertHoneycomb(AlertTask):
+    def main(self):
+        # look for events in last 10 mins
+        search_query = SearchQuery(minutes=10)
+
+        search_query.add_must([
+            TermMatch('category', 'syslog'),
+            TermMatch('processname', 'Honeycomb')
```

Comment:
Can we add a requirement that the logging priority be critical?  Honeycomb does emit other severity levels, but criticals are the only ones we're really interested in for the sake of alerting.

Reply:
Done in https://github.com/mozilla/MozDef/pull/747/commits/aff07d46fcd5496e0d36676b111ef3bd146757a5